lib/babel.js

```diff
@@ -32,20 +34,46 @@ module.exports = {
 	}
 };
 
+function send(name, data) {
+	process.send({name: name, data: data});
+}
+
+process.on('uncaughtException', function (exception) {
+	send('uncaughtException', {uncaughtException: serializeValue(exception)});
+});
+
 var transpiled = babel.transformFileSync(testPath, options);
 requireFromString(transpiled.code, testPath, {
 	appendPaths: module.paths
 });
 
 if (!avaRequired) {
-	console.error('No tests found in ' + testPath + ', make sure to import "ava" at the top of your test file');
-	setImmediate(function () {
-		process.exit(1);
-	});
+	throw new Error('No tests found in ' + testPath + ', make sure to import "ava" at the top of your test file');
 }
 
 process.on('message', function (message) {
-	if (message['ava-kill-command']) {
+	var command = message['ava-child-process-command'];
+	if (command) {
+		process.emit('ava-' + command, message.data);
+	}
+});
+
+process.on('ava-kill', function () {
+	setTimeout(function () {
 		process.exit(0);
+	}, process.env.AVA_APPVEYOR ? 100 : 0);
```

Comment:
@vdemedes I think the intention is to only do it when *we* run AVA's tests on AppVeyor, not when the user run their tests there.